### PR TITLE
Adds member superlative 

### PR DESCRIPTION
**Pull request by deadlybutter**
Also some small bug fixes and optimizations, scroll stop function for when you're doing local development 

@DFurnes

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -57,13 +64,21 @@ $(document).on('ready', function() {
...
   function updateFeaturedMembers() {
-
+    $.get('/data/featured_members.json', function(data) {
+      var members = data.members;
+      console.log(members[0]);
+      $('.member').each(function(index) {
+        var chosenMember = members[Math.floor(Math.random()*members.length)]; //Its the chosen one!
+        console.log(chosenMember);
+        members.splice(members.indexOf(chosenMember), 1);
+        $(this).find('h2').text(chosenMember.title);
```
Comment (DFurnes) [inline public/js/main.js]: This would be a great candidate for client-side templating!

File: public/js/main.js [hunk 2]
```diff
@@ -57,13 +64,21 @@ $(document).on('ready', function() {
   }
 
   function updateFeaturedMembers() {
-
+    $.get('/data/featured_members.json', function(data) {
+      var members = data.members;
+      console.log(members[0]);
+      $('.member').each(function(index) {
+        var chosenMember = members[Math.floor(Math.random()*members.length)]; //Its the chosen one!
+        console.log(chosenMember);
```
Comment (DFurnes) [inline public/js/main.js]: Sneaky `console.log`!

File: sass/main.scss [hunk 3]
```diff
@@ -99,6 +100,7 @@ img {
 }
 
 .slide_counts {
+  @extend .section;
```
Comment (DFurnes) [inline sass/main.scss]: I would recommend against using `@extend` for this. It creates really crazy selectors if you're not careful.

File: sass/main.scss [hunk 4]
```diff
@@ -82,6 +82,7 @@ img {
 }
 
 .slide_dosomething {
```
Comment (DFurnes) [inline sass/main.scss]: Our convention on the front-end team is to use dashes in class names, and BEM-style child/modifier naming. Not necessary, but could be nice to keep things consistent (and easier for on boarding if anyone else needs to jump into the codebase).

File: public/js/main.js (hunk 2 above)
Comment (deadlybutter) [inline public/js/main.js]: theres gonna be a day that I remember I left one

File: sass/main.scss (hunk 3 above)
Comment (deadlybutter) [inline sass/main.scss]: okie dokes

File: sass/main.scss [hunk 5]
```diff
@@ -225,3 +229,38 @@ img {
...
 }
+
+.slide_members {
+  @extend .section;
+
+  h1 {
+    padding-bottom: 32px;
+  }
+
+  .member {
```
Comment (DFurnes) [inline sass/main.scss]: I'd avoid nesting this rule unless you really need to. Just like extends, nesting can make unintentionally crazy selectors in the outputted CSS.

File: sass/main.scss (hunk 4 above)
Comment (deadlybutter) [inline sass/main.scss]: I can do another PR and rename everything to the same convention you guys are using :+1:

Comment (DFurnes) [inline sass/main.scss]: Sounds great!

File: sass/main.scss (hunk 5 above)
Comment (deadlybutter) [inline sass/main.scss]: Ill go remove some of the nesting where I can, just looked at the outputted css and I see what you mean

File: public/js/main.js (hunk 1 above)
Comment (deadlybutter) [inline public/js/main.js]: I'd like to get this PR deployed ASAP for Ben, is it okay if I keep this for the moment and work on adding the templating/middle where next?

**Pull request merged**